appinfo/Migrations/Version20170329194544.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace OCA\oauth2\Migrations;
+
+use Doctrine\DBAL\Exception\UniqueConstraintViolationException;
+use OCA\OAuth2\Db\Client;
+use OCA\OAuth2\Db\ClientMapper;
+use OCP\Migration\ISimpleMigration;
+use OCP\Migration\IOutput;
+
+/**
+ * This adds the default client ids for ownCloud mobile and desktop clients
+ */
+class Version20170329194544 implements ISimpleMigration {
+
+	private static $registry = [
+		['Desktop Client', 'http://localhost:*', 'xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69', 'UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh'],
+		['Android', 'oc://android.owncloud.com', 'e4rAsNUSIUs0lF4nbv9FmCeUkTlV9GdgTLDH1b5uie7syb90SzEVrbN7HIpmWJeD', 'dInFYGV33xKzhbRmpqQltYNdfLdJIfJ9L5ISoKhNoT9qZftpdWSP71VrpGR9pmoD'],
```

Comment:
oc:// is actually a very short name.  It it not true that these need to be globally registered on the whole phone?
Would it not be better to have something longer such as   owncloud://  or even owncloudauth://  ?

Also, can two different owncloud client co-exist on the same phone?   (say, a branded client, the original owncloud client, and the nextcloud client)